src/main/scala/ch/openolitor/stammdaten/StammdatenDeleteService.scala

```diff
@@ -220,11 +220,19 @@ class StammdatenDeleteService(override val sysConfig: SystemConfig) extends Even
     }
   }
 
+  private def deleteKoerbeForDeletedLieferung(lieferung: Lieferung)(implicit personId: PersonId, session: DBSession, publisher: EventPublisher) = {
+    logger.debug(s"deleteKoerbeForDeletedLieferung => lieferung: $lieferung")
```

Comment:
I have the feeling that it would be clearer if this function is inlined. Or have a specialized function in the Repository "deleteKoerbeForLieferung"

Reply:
Actually you are right, and there is already a function in the deletestammdaten. I'll replace it